common/version.go

```diff
@@ -39,7 +39,7 @@ import (
 	"strings"
 )
 
-const Blobfuse2ListContainerURL = "https://blobfuse2.blob.core.windows.net/release"
+const Blobfuse2ListContainerURL = "https://blobfuse2.z13.web.core.windows.net/release"
```

Comment:
Can we make a short URL for this. Also please add a comment what this short URL will have for reference.

Reply:
does not work as the aka.ms link needs exact url destination and we suffix the url with the version numbers at times.